tools/build/Makefile-common.in

```diff
@@ -81,14 +81,22 @@ COMMON_NQP_SOURCES  = \
 
 NQP_COMBINED = NQP.nqp
 
+// No QAST_SOURCES, just one file, no need to combine
```

Comment:
I dind't know // worked as comment syntax in Makefiles. I've certainly never seen it used before. Are you sure that's compatible with alle relevant make implementations?